internal/receiver/databricksreceiver/metadata.yaml

```diff
@@ -1,13 +1,36 @@
 name: databricksreceiver
-attributes:
+resource_attributes:
   databricks.instance.name:
     description: The name of the Databricks instance as defined by the value of the "instance_name" field in the config
+    type: string
+attributes:
   job_id:
     description: The numeric ID of the Databricks job
+    type: int
   task_id:
     description: The name of the Databricks task
+    type: string
+  cluster_id:
+    description: n/a
```

Comment:
Thanks, I've added descriptions to these attributes. Will commit this along with changes you suggested relating to moving attributes from metrics to resources, in the next couple of business days. There are still lots of `n/a`s in the metadata metric descriptions, but mostly because Spark doesn't provide descriptions themselves (or at least I haven't been able to find any).